modules/list_authors.rb

```diff
@@ -0,0 +1,12 @@
+module AuthorList
+  def list_all_authors
+    if @authors.length.zero?
+      puts 'No authors are available'
+    else
```

Comment:
- [ ] **Required:** Authors are never created in your app. Kindly link the `create Author` to your `create game.`

- [ ] **Required:** Because they are never created as stated above, they can not be listed. Kindly fix both scenarios.